### ISSUE TITLE
Feedbacks for DTbook to Epub3 accessibility report with Ace

### ISSUE THREAD

**Issue by NPavie** [daisy/pipeline#752]
With DAISY Pipeline App version: 1.2.7-RC3,  Engine version: 1.14.16, on Windows 11.

I did some conversion from dtbook to epub3 and got some feedbacks from validation with Ace : 

- [x] The package.opf file is missing a lang attribute (`xml:lang` was not set in my dtbook, but was indicated in the script parameter)
- [x] The package.opf had the following namespaces included in the result : 
`xmlns:f="http://www.daisy.org/ns/pipeline/internal-functions" xmlns:pf="http://www.daisy.org/ns/pipeline/functions" xmlns:rdfa="rdfa-functions"`
- [x] The package.opf reference a non-existing file : `<link rel="record" href="zedai-mods.xml" media-type="application/mods+xml"></link>`
- [ ] `epub:type` attributes are not associated with corresponding aria `role` attributes when possible (see the [mapping table](https://idpf.github.io/epub-guides/epub-aria-authoring/#sec-mappings)) :
  - [x] notes are created as aside but without attribute `epub:type="footnote" role="doc-footnote"`
  - [ ] notes references are converted to span instead of anchor pointing to the note content with attribute `epub:type="noteref" role="doc-noteref"`
  - [x] Ace/aXe reports a missing ` role="doc-toc"` on `<nav epub:type="toc">`
- [x] Ace reports missing metadata `schema:accessibilityHazard`, `schema:accessMode`, `schema:accessibilityFeature`, `schema:accessModeSufficient`, `schema:accessibilitySummary`,

**Comment by bertfrees** [comment]
All of these issues except the 4th one (`epub:type` attributes are not associated with corresponding aria role attributes when possible) have been fixed in https://github.com/daisy/pipeline-modules/commit/efb1b61b38395b33a5d0a942d27a761d8ffd95c4.

**Comment by bertfrees** [comment]
The remaining issue is now also fixed. I couldn't reproduce this part though:

> notes references are converted to span instead of anchor pointing to the note content with attribute epub:type="noteref" role="doc-noteref"

@NPavie Let me know if this still doesn't work for you in the next version, with an example.
